### PR TITLE
Update gen-jwt.py

### PR DESCRIPTION
**Pull request by sun-wenming**
I use python 3.7.3.  This is hava error: `TypeError: from_buffer() cannot return the address of a unicode object`
Solve by change to bytes.

### CODE REVIEW

File: security/tools/jwt/samples/gen-jwt.py [hunk 1]
```diff
@@ -28,9 +28,10 @@
 def main(args):
     """Generates a signed JSON Web Token from local private key."""
     with open(args.key) as f:
-        pem_data = f.read()
+        pem_data_encode = f.read()
     f.closed
 
+    pem_data = pem_data.encode("utf-8")
```
Comment (howardjohn) [inline security/tools/jwt/samples/gen-jwt.py]: will this break python 2.x now? Not sure if this is a concern though

Comment (sun-wenming) [reply security/tools/jwt/samples/gen-jwt.py]: I am test with Python 2.7.16 is ok.But I am not sure about the previous 2.x version, sorry.

Comment (sun-wenming) [reply security/tools/jwt/samples/gen-jwt.py]: There are some variable name errors that have been modified.
latest：
```
"""Generates a signed JSON Web Token from local private key."""
    with open(args.key) as f:
        pem_data = f.read()
    f.closed

    pem_data_encode = pem_data.encode("utf-8")
    key = jwk.JWK.from_pem(pem_data_encode)
```

**Comment by howardjohn** [comment]
/ok-to-test
@sun-wenming you need to sign the CLA for us to approve this

**Comment by sun-wenming** [comment]
I signed it!

**Pull request merged**